backend/GoogleAPI.ts

```diff
@@ -24,16 +28,49 @@ export async function sendFeedbackToGoogleSheets({
   feedbackText: string;
   userDetails: User;
 }) {
-  const date = `${new Date().toLocaleDateString('fi-FI')} ${new Date().toTimeString()}`;
-  const feedbackGiverDetails = `${userDetails.firstName} ${userDetails.lastName}\n${userDetails.email}`;
-  await SHEETS.spreadsheets.values.append({
-    auth: AUTHENTICATION,
-    spreadsheetId: SPREADSHEET_ID,
-    range: `Palautteet!A:C`,
-    requestBody: {
-      majorDimension: 'ROWS',
-      values: [[feedbackText, feedbackGiverDetails, date]],
-    },
-    valueInputOption: 'USER_ENTERED',
-  });
+  try {
+    const credentialsEnvParse = process.env.GOOGLE_SERVICE_ACCOUNT_CREDENTIALS
+      ? jwtInputSchema.safeParse(
+          JSON.parse(process.env.GOOGLE_SERVICE_ACCOUNT_CREDENTIALS),
+        )
+      : null;
+
+    if (credentialsEnvParse?.success === false) {
+      console.error(
+        'Environment variable: GOOGLE_SERVICE_ACCOUNT_CREDENTIALS is invalid!',
+      );
+    }
```

Comment:
Jos käytössä onkin `env`-variable ja se on virheellinen, printataan syy miksi näin kävi. Jos taas käytössä on `JSON`-tiedosto tämä kohta ohitetaan

Sitten kun otetaan lokitiedot käyttöön, niin tästä lienee hyvä lisätä logeihin tieto